ui/admin/tests/acceptance/credential-store/credentials/create-test.js

```diff
@@ -300,32 +315,33 @@ module(
         );
       });
 
-      await click(`[href="${urls.newCredential}"]`);
-      await click('[value="json"]');
-      await click('[type="submit"]');
+      await click(commonSelectors.HREF(urls.newCredential));
+      await click(selectors.FIELD_TYPE_JSON);
+      await click(commonSelectors.SAVE_BTN);
 
-      assert
-        .dom(commonSelectors.ALERT_TOAST_BODY)
-        .hasText('Error in provided request.');
+      assert.dom(commonSelectors.ALERT_TOAST_BODY).hasText(errorMessage);
     });
 
     test('cannot navigate to json credential when feature is disabled', async function (assert) {
       await visit(urls.credentials);
 
-      await click(`[href="${urls.newCredential}"]`);
+      await click(commonSelectors.HREF(urls.newCredential));
+
       assert.false(featuresService.isEnabled('json-credentials'));
       assert.true(
         instances.staticCredentialStore.authorized_collection_actions.credentials.includes(
           'create',
         ),
       );
-      assert.dom('.hds-form-radio-card').exists({ count: 2 });
       assert
```

Comment:
Refactored these assertions to not use hds specific classes and reusing the existing selectors.

- We don't check anymore there are 2 `hds-form-radio-card`. Instead we check username & password and SSH key are present but JSON being not present.
- We kept the `hasAttribute` assertion, but I personally think can be redundant since the selector itself its using `value` i.e: `export const FIELD_TYPE_USERNAME_PASSWORD = '[value=username_password]';`